interp/constrexpr.mli

```diff
@@ -45,10 +45,17 @@ type entry_relative_level = LevelLt of entry_level | LevelLe of entry_level | Le
 type notation_entry = InConstrEntry | InCustomEntry of string
 
 (* A notation entry with the level where the notation lives *)
-type notation_entry_level = notation_entry * entry_level
+type notation_entry_level = {
+  notation_entry : notation_entry;
+  notation_level : entry_level;
+}
```

Comment:
@herbelin if you have the energy, the form:
```
module Entry_level = struct
  type t = { entry : notation_entry;
                 ; level : entry_level
   }
end
```
tends to work better.

Reply:
I don't buy the "; ..." formatting which moreover is not the  most "standard" way to format records in the code of Coq.

Otherwise, what are the pros and cons of using a module? And how do you measure the "better"?

Reply:
Using a module only makes sense if you have functions directly related to it together. If it is just a one off type its probably not worth it.